src/bundles/sounds/riffwave.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable */
```

Comment:
Similar to the previous one, please avoid disabling eslint and prettier totally. Try to conform to the rules set if possible, if not add `// eslint-disable-next-line rule-to-disable` to disable the specific rule for the next line.

Ill avoid pointing this out for future files, but please do the same for the rest.